crates/rolldown_binding/src/options/plugin/config/binding_manifest_plugin_config.rs

```diff
@@ -11,7 +11,7 @@ pub struct BindingManifestPluginConfig {
   pub out_path: String,
   #[napi(ts_type = "() => boolean")]
   pub is_legacy: Option<JsCallback<(), bool>>,
-  #[napi(ts_type = "() => Map<string, string>")]
+  #[napi(ts_type = "() => Record<string, string>")]
```

Comment:
Do we still need the `ts_type`?
```suggestion
  #[napi]
```

Reply:
Because `napi-rs` doesn't correctly parse `JsCallback`, we need to explicitly annotate the function type here.